src/scripts.js

```diff
@@ -125,7 +125,6 @@ function viewFavorites() {
     createRecipeCards(user.favoriteRecipes);
   }
   checkFavoriteActive();
-  // checkRecipeToCookActive();
```

Comment:
For the viewFavorites function, do we not need to invoke checkRecipeToCookActive here too so that it displays the correct format of these recipes?